### PR TITLE
converted LoggerTest to Kotlin and...

### PR DESCRIPTION
**Pull request by Jessica-Kuo**
converted LoggerTest to Kotlin and updated Gradle files to include Kotlin support.

LoggerTest.java:
LoggerTest-java

LoggerTest.kt:
LoggerTest-kotlin

### CODE REVIEW

File: analytics-samples/analytics-wear-sample/build.gradle [hunk 1]
```diff
@@ -17,4 +18,10 @@ dependencies {
   implementation project(':analytics')
   implementation project(':analytics-wear')
   implementation 'androidx.wear:wear:1.0.0'
+    implementation "androidx.core:core-ktx:+"
```
Comment (prayansh) [inline analytics-samples/analytics-wear-sample/build.gradle:21]: can we remove this whitespace

File: analytics-tests/build.gradle [hunk 2]
```diff
@@ -1,10 +1,17 @@
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
 
 apply from: rootProject.file('gradle/android.gradle')
 
 dependencies {
   api project(':analytics')
+    implementation "androidx.core:core-ktx:+"
```
Comment (prayansh) [inline analytics-tests/build.gradle:8]: same whitespace removal

File: analytics-wear/build.gradle [hunk 3]
```diff
@@ -8,6 +9,12 @@ dependencies {
   // TODO: we should update this dependency.
   //noinspection GradleDependency
   api 'com.google.android.gms:play-services-wearable:10.2.6'
+    implementation "androidx.core:core-ktx:+"
```
Comment (prayansh) [inline analytics-wear/build.gradle:12]: same whitespace removal

File: build.gradle [hunk 4]
```diff
@@ -1,18 +1,21 @@
 buildscript {
-  repositories {
+    ext.kotlin_version = '1.4.0-rc'
+    repositories {
     mavenCentral()
```
Comment (prayansh) [inline build.gradle:4]: can we actually fix the formatting here a bit

**Comment by prayansh** [comment]
Just realized one more thing: Do we need to make changes to the following files?
analytics-samples/analytics-wear-sample/build.gradle
analytics-tests/build.gradle
analytics-wear/build.gradle

Since the kotlin change is isolated to the `analytics` module, introducing the kotlin boilerplate to the other files doesnt make much sense here

**Comment by prayansh** [comment]
also seems like the CI is failing on some lint. You might wanna try running this command in the root folder
`./gradlew spotlessApply` to run the linter

**Comment by Jessica-Kuo** [comment]
Oh! You're right, those shouldn't be necessary. Android Studio added them automatically but I can take them out and see if the build still runs.

**Pull request merged**